packages/sit-onyx/src/styles/root.scss

```diff
@@ -18,6 +18,15 @@
     --onyx-shadow-soft-right: 0.5rem 0 1.5rem 0 rgba(0, 0, 0, 0.1);
   }
 
+  :root,
+  .onyx-light {
```

Comment:
Imho we can drop support for the `.onyx-light` and `.onyx-dark` class since I don't see the benefit of them

Reply:
there is no use case where inside a dark page, a light section is needed, right?